### ISSUE TITLE
Qt warnigs on  qt/transactiongraphwidget.cpp

### ISSUE THREAD

**Issue by sickpig** [gandrewstone/bitcoin#1992]
It seems to be related to #1935, cc: @Justaphf 
```
../../src/qt/transactiongraphwidget.cpp: In member function ‘void TransactionGraphWidget::setTransactionsPerSecond(double, double, double)’:
../../src/qt/transactiongraphwidget.cpp:298:22: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     if (nCurrentTime >= nLastRedrawTime + nRedrawRateMillis)
         ~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../src/qt/transactiongraphwidget.cpp: In member function ‘void TransactionGraphWidget::updateTransactionsPerSecondLabelValues()’:
../../src/qt/transactiongraphwidget.cpp:320:15: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
         if (i <= numSamples)
             ~~^~~~~~~~~~~~~
../../src/qt/transactiongraphwidget.cpp:326:19: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
             if (i <= numSamples)
                 ~~^~~~~~~~~~~~~
../../src/qt/transactiongraphwidget.cpp:355:15: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
         if (i <= numSamples)
             ~~^~~~~~~~~~~~~
../../src/qt/transactiongraphwidget.cpp:361:19: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
             if (i <= numSamples)
                 ~~^~~~~~~~~~~~~
```

**Comment by Justaphf** [comment]
@sickpig Thanks for the report, looking into this tonight, hopefully have a PR in by the weekend.

**Comment by sickpig** [comment]
fixed by #2011
